packages/clipboard/src/CopyCodeButton.tsx

```diff
@@ -8,19 +10,27 @@ import styles from './styles/CopyCodeButton.module.css';
 type CopyCodeButtonProps = ButtonHTMLAttributes<HTMLButtonElement> & {
   children: ReactNode;
   'data-test-id'?: string;
+  variant?: CopyToClipboardProps['variant'];
 };
 
 const CopyCodeButton = forwardRef<HTMLButtonElement, CopyCodeButtonProps>(
-  ({ className, children, 'data-test-id': testId = 'copy-code-button', ...rest }, ref) => {
+  ({ className, children, 'data-test-id': testId = 'copy-code-button', variant, ...rest }, ref) => {
     return (
       <button
         ref={ref}
         data-test-id={testId}
         type="button"
-        className={cx(styles['CopyCodeButton'], className)}
+        className={cx(styles['CopyCodeButton'], className, variant && styles[variant])}
         {...rest}
       >
         {children}
+        {variant && (
```

Comment:
As a second part of the first suggestion, can we then check here if kind === 'basic' || kind === 'minimal'? I like if we can make this a little more explicit.